loadtest/store/store.go

```diff
@@ -18,6 +18,8 @@ type UserStore interface {
 	Config() model.Config
 	// Channels return the channels for a team.
 	Channels(teamId string) ([]model.Channel, error)
+	// ChannelMember returns the ChannelMember for the given channelId and userId.
+	ChannelMember(channelId, userId string) (*model.ChannelMember, error)
```

Comment:
By moving this to the `UserStore` we should make it return a value instead of a pointer.